### DISCUSSION TITLE
How to enable all new users access to an app by default?

### DISCUSSION THREAD

**Discussion by xlog-null** [Budibase/budibase#8174] [Help] [answered]
Hi, I have budibase setup with SSO using company email. I need 200+ people to be able to access the app.

In this moment what happens is that if a user has never used budibase, they're not in the user database. When they sign in with SSO, they're added automatically which is great. But then I have to go to the individual user and assign them rights specifically to use the app. Which is not manageable at all...

Even if beforehand, I import the user by csv import with their details to my app specifically, it still doesn't work, and I have to manually add them.

Also a side note- I can't remove users already added to my app..? There is no option anywhere. So to test my csv import I imported the same user three times and now I can't remove the entries:
![image](https://user-images.githubusercontent.com/92792093/194506596-95eb4370-750f-45d7-89cd-fa0d11742a9d.png)

Thanks!

**Comment by melohagan** [comment, marked as answer]
Hey @xlog-null 

You should be able to remove users from the admin user management section: https://docs.budibase.com/docs/user-management#deleting-a-user

You can assign users to groups, which will give them access to apps upon invitation. 
Groups are available in the Pro plan: https://budibase.com/pricing/
